findimports.py

```diff
@@ -1005,6 +1019,8 @@ def main(argv=None):
         args = parser.parse_args(args=argv[1:] if argv else None)
     except SystemExit as e:
         return e.code
+    if args.condense_to_packages and args.condense_to_packages_externals:
+        raise argparse.ArgumentError('only one of -p and -pE can be provided')
```

Comment:
I think this will print a traceback at the user instead of reporting an error message?

Reply:
It does report the message, but there was a bug, message is 2nd arg, which is mandatory.